tests/conftest.py

```diff
@@ -34,7 +19,7 @@ def slices():
         c_fpath = Path(f"{input_dir}/slice{i}_coor.csv")
 
         # Create ann data object of each slice and add other properties
-        _slice = sc.read_csv(s_fpath)
+        _slice = sc.read_csv(s_fpath, dtype="float64")
```

Comment:
needed for correct dtype operations in the code (NumpyBackend hid away this discrepancy for us initially).